first.css

```diff
@@ -0,0 +1,112 @@
+body {
+  height: 7000px;
+}
+
+.floating-subscribe-wrapper 
+{
+    background: #fff;
```

Comment:
Check what are CSS variables and let's use those for colors - like primary color, secondary colors